lib/internal/webstreams/util.js

```diff
@@ -129,21 +131,32 @@ function transferArrayBuffer(buffer) {
   return res;
 }
 
-function isArrayBufferDetached(buffer) {
+function isDetachedBuffer(buffer) {
+  if (!isArrayBuffer(buffer))
+    throw new ERR_INVALID_ARG_TYPE('buffer', 'ArrayBuffer', buffer);
```

Comment:
Indeed, it looks better to follow the coding convention. 

I updated the util using `assert` instead, since, as you mentioned, either `true` or `false` won't make sense if a provided value is not a buffer. We can guarantee a passing value is a buffer since it's internal util. PTAL.